client/src/js/SM/Manage.js

```diff
@@ -3436,7 +3444,7 @@ SM.Manage.Asset.LabelField = Ext.extend(Ext.form.Field, {
       data: [],
     })
     const config = {
-      name: 'labelIds'
+      name: 'labelNames'
```

Comment:
Changing the field's name here causes the field to be skipped during the call to assetPropsFormPanel.getForm().setValues(apiAsset) on line 2928. The apiAsset is returned from a GET which does not have a labelNames property. So this field's setValue() function is not called and the labels currently mapped to an asset are not displayed in the pop-up.